guardian/Makefile

```
@@ -96,6 +96,8 @@ ut:
 ## Run all CI steps for build and test, likely other targets.
 ci: static-checks ut
 
+clean:
+
```

I image at the very least, you will want this.

```suggestion
clean:
	# Clean .created files which indicate images / releases have been built.
	find . -name '.*.created*' -type f -delete
	find . -name '.*.published*' -type f -delete
	rm -rf bin

```